polyfills/URL/tests.js

```diff
@@ -1,28 +1,35 @@
 /* eslint-env mocha, browser*/
 /* global proclaim, it */
 
+function toArray(it) {
```

Comment:
I wanted to make as few changes to the tests as possible.  I don't really mind either of those options, or just leaving it as-is.